content/blogposts/2023/c-cpp-cross-repo.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Cross-repository precise code navigation for C and C++'
+authors:
+  - name: Varun Gandhi
+    url: https://github.com/varungandhi-src
+publishDate: 2023-06-12T10:00-07:00
+description: 'Sourcegraph now supports cross-repository precise code navigation for C and C++ for code indexed by scip-clang.'
+tags: [blog]
+slug: 'c-cpp-cross-repo'
+published: true
+---
+
+We're excited to announce cross-repository precise code navigation support
+for C and C++ in [scip-clang](https://sourcegraph.com/github.com/sourcegraph/scip-clang).
+This feature addition brings scip-clang up to parity with our other SCIP indexers,
```

Comment:
@varungandhi-src You could fit in a short mention about the original blog post here. Here's a suggestion on how you could do it without distracting too much from the content of this post.

```suggestion
We [initially released scip-clang last month](https://about.sourcegraph.com/blog/announcing-scip-clang),  and this feature addition brings scip-clang up to parity with our other SCIP indexers,
```

Reply:
I've added a mention for the announcement blog post.